academy/4-my-own-chain-exer/game-fifo-exer.md

```diff
@@ -25,15 +25,15 @@ Players can now reject games, so there is a way for stale games to be removed fr
 * What Ignite CLI commands, if any, will assist you?
 * How do you adjust what Ignite CLI created for you?
 * How would you unit-test these new elements?
-* How would you use Ignite CLI to run locally a one-node blockchain and interact with it via the CLI and see what you get?
+* How would you use Ignite CLI to locally run a one-node blockchain and interact with it via the CLI to see what you get?
 
 For now, don't bother yet with future ideas like wager handling.
 
 ## Next up
 
-When you are satisfied, or not, with what you did, have a look at a solution, which includes:
+When you are satisfied (or not) with what you have, look at some solutions which relate to:
 
-* a [FIFO](../4-my-own-chain/game-fifo.md),
```

Comment:
I made these changes to the link texts to "hide" the specific clues to the solutions that using "FIFO" etc represent - should be checked to make sure my alternatives are reasonable substitutes.

Reply:
Here the FIFO is more about system performance. So "Secondary game structure for performance"?

Reply:
I would distill down to "System performance" maybe

Reply:
Ok